meta/2025-01-15.114_for_115/rs8t_tmol

```diff
@@ -20,9 +20,14 @@ species.common_name	Darkling ground beetles
 #CONF	ASM_URL	https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/907/166/875/GCA_907166875.3_Tenebrio_molitor_v3
 
 # Community GFF3 loading
-#no CONF	DATA_INIT	zcat  | cat > fixed.gff3
-#no CONF	GFF_FILE	fixed.gff3
-#no CONF	PEP_FILE	pep.faa
+#CONF	DATA_INIT	zcat GCA_907166875.3_Tenebrio_molitor_v3_genomic.gff.gz | grep -vFw ID=exon-TMOL_6.224.1-1 | perl -pe 's/Parent=gene-/Parent=rna-/ if (m/\tCDS\t/ && m/ID=cds-CAH1374853.1;Parent=gene-TMOL_6.224.1;/)' > pre_fixed.gff3
```

Comment:
do you want to provide more details on what's being fixed? upi to you though

Reply:
Will try to remember the details and add a comment, yes.